app/components/workflow-default-dialog.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+
+const WorkflowDefaultDialog = ({ onSuccess }) => (
+  <div>
+    <Translate content="workflowDefaultDialog.text" />
```

Comment:
I'm not sure if this needs to be translated. The admin pages are only for Zooniverse team members.